contracts/interfaces/ITablelandTables.sol

```diff
@@ -63,6 +63,14 @@ interface ITablelandTables {
      */
     event SetController(uint256 tableId, address controller);
 
+    /**
+     * @dev Struct containing parameters needed to run a sql statement: tableId and statement.
+     */
+    struct Statement {
+        uint256 tableId;
```

Comment:
I think we should add descriptions tableId and statement, then you could remove ": tableId and statement" from the main doc string